projects/ion/src/lib/skeleton/skeleton.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/angular';
+
+import { IonSkeletonComponent } from './skeleton.component';
+
+const defaultProps = {
+  variant: 'rect',
+  width: 50,
+  height: 50,
+} as IonSkeletonComponent;
+
+const rectangularProps = {
+  variant: 'rect',
+  width: 400,
+  height: 100,
+} as IonSkeletonComponent;
+
+const circularProps = {
+  variant: 'circular',
+  width: 50,
+  height: 50,
+} as IonSkeletonComponent;
+
+const sut = async (customProps: IonSkeletonComponent): Promise<HTMLElement> => {
```

Comment:
```suggestion
const sut = async (customProps: IonSkeletonProps): Promise<HTMLElement> => {
```
I guess the correct type of custom props here is IonSkeletonProps too.